### PR TITLE
db-testing - added: test coverage with jest, redis, and fleshed out api

### PR DESCRIPTION
**Pull request by LoganAHart**

### CODE REVIEW

File: db-cassandra/index.js [hunk 1]
```diff
@@ -49,42 +36,43 @@ const validCols = {
...
-  if (rest_id !== rest_id) {
-    throw new Error('bad request: 400');
-    return 'error: invalid rest_id';
+  const paramsID = req.params.restaurantId
+  const rest_id = parseInt(paramsID, 10);
+  return reg.test(paramsID) ? rest_id : 'bad request: 400';
+};
+
+const checkParamSelect = (req) => {
+  const paramsID = req.params.restaurantId
```
Comment (tonaramos) [inline db-cassandra/index.js:48]: ; at the end.

File: redis-cache/redisCache.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+});
+
+const closeInstance = (callback) => {
+  client.quit(callback)
+}
+
+module.exports = {
+  client,
+  closeInstance,
+};
```
Comment (tonaramos) [inline redis-cache/redisCache.js:21]: New line at the end.

File: package.json [hunk 3]
```diff
@@ -27,15 +28,18 @@
     "axios": "^0.18.0",
     "body-parser": "^1.18.3",
     "cassandra-driver": "^3.5.0",
+    "chai": "^4.1.2",
```
Comment (tonaramos) [inline package.json:31]: Are you testing with mocha, chai, and stambul? If not we can get rid of these before deployment.

**Review by tonaramos — APPROVED** [review]
Looks good.

**Comment by LoganAHart** [comment]
Thanks for the review/notes!

**Pull request merged**